src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -59,6 +60,16 @@
 #endif
 #include "proto/capabilities.pb.h"
 
+/**
+ * @def OTBR_CONFIG_BORDER_AGENT_MESHCOP_EPHEMERAL_UDP_PORT
+ *
+ * Specifies the border agent UDP port for ephemeral meshcop service.
+ * If zero, an ephemeral port will be used.
```

Comment:
```suggestion
 * Specifies the border agent UDP port for meshcop-e service.
 * If zero, an ephemeral port will be used.
```

Try to avoid ambiguity.

Reply:
To be precise, it's "_meshcop-e"

Reply:
Accept, thanks.